### PR TITLE
add team counter to home page

### LINKED ISSUE #2141: Add Team counter to Home page
image
Since the "Inventory" section in the homepage is an entry point for the main subpages, it may be a good idea to add a team counter. Since we are focused more towards teams, it may make sense to add the most 3 recently edited teams in here as well.

### PR DESCRIPTION
**Pull request by frzyc**
## Issue or discord link

- Resolves #2141

## Testing/validation

Screenshot 2024-05-18 002549

## Checklist before requesting a review (leave this PR as draft if any part of this list is not done.)

- [ ] I have commented my code in hard-to understand areas.
- [ ] I have made corresponding changes to README or wiki.
- [ ] For front-end changes, I have updated the corresponding English translations.
- [ ] I have run `yarn run mini-ci` locally to validate format and lint.
- [ ] If I have added a new library or app, I have updated the deployment scripts to ignore changes as needed

### CODE REVIEW

File: libs/gi/page-home/src/InventoryCard.tsx [hunk 1]
```diff
@@ -183,6 +204,67 @@ export default function InventoryCard() {
...
+                </strong>
+              }
+            />
+            <Chip
+              sx={{ flexGrow: 1, cursor: 'pointer' }}
+              color={numBuilds ? 'success' : 'secondary'}
+              icon={<BuildIcon />}
+              label={
+                <strong>
+                  {t(`ui:tabs.teams`)} {numBuilds}
```
Comment (nguyentvan7) [inline libs/gi/page-home/src/InventoryCard.tsx:245]: String is wrong. Builds, not teams

File: libs/gi/page-home/src/InventoryCard.tsx [hunk 2]
```diff
@@ -183,6 +204,67 @@ export default function InventoryCard() {
...
+              icon={<LoadoutIcon />}
+              label={
+                <strong>
+                  {t(`ui:tabs.loadouts`)} {numLoadout}
+                </strong>
+              }
+            />
+            <Chip
+              sx={{ flexGrow: 1, cursor: 'pointer' }}
+              color={numBuilds ? 'success' : 'secondary'}
```
Comment (nguyentvan7) [inline libs/gi/page-home/src/InventoryCard.tsx:241]: Should we start using less colors here? Or are you gonna do that in a different pr

**Review by nguyentvan7 — CHANGES_REQUESTED** [review]

File: libs/gi/page-home/src/InventoryCard.tsx (hunk 2 above)
Comment (frzyc) [reply libs/gi/page-home/src/InventoryCard.tsx:241]: probably a different PR, will def need more design guidance before pulling the trigger on that.

File: libs/gi/localization/assets/locales/en/ui.json [hunk 3]
```diff
@@ -10,7 +10,9 @@
     "setting": "Settings",
     "doc": "Docs",
     "scanner": "Scanners",
-    "archive": "Archive"
+    "archive": "Archive",
+    "loadouts": "Loadouts",
+    "builds": "Builds"
```
Comment (nguyentvan7) [inline libs/gi/localization/assets/locales/en/ui.json:15]: Technically not a tab, but I'll allow it

**Review by nguyentvan7 — APPROVED** [review]
